reference/conanfile/running_and_output.rst

```diff
@@ -30,6 +30,10 @@ so running with ``-vwarning` will output calls to ``warning()`` and ``error()``,
 
 Note that these methods return the output object again, so that you can chain output calls if needed.
 
+Using the ``core:warnings_as_errors`` conf, you can make Conan raise an exception when either a warning or error is printed.
```

Comment:
I am missing some example of syntax like ``core:warning_as_errors=["network", "..."]`` (is it a list?)

Reply:
Yeah, it is not very clear, the ``skip_warnings`` is another config, it is not evident what format ``warnings_as_errors`` gets, is it a boolean?. It is a new ``conf`` with its own values.

Reply:
I think that now that it is a list, it really deserves a bit of explanation and example